module/prover/l1/config.go

```diff
@@ -106,6 +108,11 @@ func (prc *ProverConfig) getForkParameters() *lctypes.ForkParameters {
 					Epoch:   364032,
 					Spec:    &ElectraSpec,
 				},
+				{
+					Version: []byte{6, 0, 0, 0},
+					Epoch:   411392,
```

Comment:
@yoshidan Where did `411392` come from? As for today, the consensus spec and implementations (I checked lodestar and lighthouse) have not set a concrete value to the Fulu HF epoch.